types/lhr/lhr.d.ts

```diff
@@ -197,6 +197,8 @@ declare module Result {
 
   /** Gather mode used to collect artifacts. */
   type GatherMode = 'navigation'|'timespan'|'snapshot';
+
+  type MetricAcronym = 'FCP' | 'LCP' | 'TBT' | 'CLS' | 'INP' | 'SI' | 'TTI' | 'FMP';
```

Comment:
[Over generalizing for fun](https://www.typescriptlang.org/play?#code/C4TwDgpgBAMhzAgJygXigcgIIYD4YCE8MBhYgEWIFFiAxYgcWIAliBJYgKWIGliZiAWWIA5YgHliABWIBFYgCViAZWIAVYgFViANWIB1YgA1iATWIAtDAG4AsAChQkKFgDGSAPYA7EAFs0UAAGACQA3nAIyAC+YRGISDHh8PFRgXb2Dq7eAM7AUACG7t5+AFwuRT7+6KQwqtZAA)